server/lib/logging/logger.ts

```diff
@@ -8,5 +8,6 @@ export function getLoggerOptions() {
     level: process.env.NODE_ENV === 'production' ? 'info' : 'debug',
     genReqId: cuid,
     serializers: stdSerializers,
+    prettyPrint: { colorize: true, translateTime: true },
```

Comment:
I think if you do things this way you're likely to break the stuff we do on the real servers (and I think this probably breaks the database logging too?). Would prefer to keep this as it is